### ISSUE TITLE
flycheck.org ownership

### ISSUE THREAD

**Issue by lunaryorn** [flycheck/flycheck#1257]
Hello,

I still own flycheck.org.  The domain is due to expire in Nov 2017, and I do not plan to renew it.  I've left Emacs and Flycheck (see #1177) and do not plan to return, so I'll no longer assume technical or legal responsibility for what's hosted on flycheck.org.

If any @flycheck/maintainers is interested in taking over I'll transfer the domain, at your own expense.  In other words, I'll not cover any expenses regarding the domain.

**Comment by mgudemann** [comment]
roughly about how much money do we talk here? I do not have a lot of time I can invest, but I think the domain should stay.

**Comment by lunaryorn** [comment]
@mgudemann Depends on what your domain hoster would charge for an .org domain.  I don't remember exactly but I think I paid around 20€ per year.

**Comment by mgudemann** [comment]
ok, I do volunteer to finance this for a while to prevent expiration, I
think it's important to stay available.
how should we discuss / manage the details ? I'll also check with my domain
hoster.

**Comment by lunaryorn** [comment]
@mgudemann Thanks.  We can sort out the details via mail or chat or whatever suits you best, but I'd like to wait for other members to have their say :)  Maybe there's a better solution to have the domain hosted by some foundation or so rather than an individual, I don't know.

**Comment by mgudemann** [comment]
@lunaryorn no problem, take your time, as you said there's still plenty of time left
if nothing else works out, just ping me

**Comment by lunaryorn** [comment]
@flycheck/maintainers Anyone else?  /cc @cpitclaudel @fmdkdd

**Comment by fmdkdd** [comment]
@lunaryorn How would that foundation thing work?  Any idea of the kind of setup that would involves?

**Comment by lunaryorn** [comment]
I don't know.

**Comment by fmdkdd** [comment]
Okay then.  I'm fine with an individual owning it.  If @mgudemann wants to register it, I don't see why not.

The domain only links to the manual, right?  Or there are any other uses (email addresses?) for it right now?

**Comment by lunaryorn** [comment]
Its just the manual

**Comment by lunaryorn** [comment]
@mgudemann I think this issue had sufficient time for comment, so since none else stepped up, I'd like to transfer flycheck.org to you some time soon, perhaps this weekend.  I think transfer authcodes have limited validity of 24h or so, so we'd need to coordinate a little bit.

Would you send me a mail?

/cc @fmdkdd @cpitclaudel

**Comment by mgudemann** [comment]
FYI, @fmdkdd @cpitclaudel you probably got the same email, Readthedocs announced that ads will be shown in the *alabaster* theme, too

**Comment by fmdkdd** [comment]
@mgudemann I saw that yes.  That's what got me looking into the readthedocs admin panel, and resulted in #1314 .

I don't like ads, but readthedocs is not running a charity.  If we don't want ads on `www.flycheck.org`, we can either pay for it (they have a "gold subscription"), or host the manual elsewhere.  I guess the main value of readthedocs is building and hosting several versions of the manual at once.

Note that we already opted out of "paid advertising", so when ads will come, we should only be seeing what they call "community ads": ads for open source projects or conferences.  This is an acceptable compromise for me.
